collect_app/src/main/java/org/odk/collect/android/utilities/QRCodeUtils.java

```diff
@@ -36,48 +37,40 @@
 import com.google.zxing.qrcode.QRCodeWriter;
 import com.google.zxing.qrcode.decoder.ErrorCorrectionLevel;
 
-import org.json.JSONException;
 import org.odk.collect.android.R;
 import org.odk.collect.android.application.Collect;
 
 import java.io.File;
-import java.io.FileOutputStream;
 import java.io.IOException;
+import java.security.MessageDigest;
+import java.util.Arrays;
 import java.util.Collection;
 import java.util.EnumMap;
 import java.util.HashMap;
 import java.util.Map;
+import java.util.zip.DataFormatException;
 
+import io.reactivex.Observable;
 import timber.log.Timber;
 
-/**
- * Created by shobhit on 13/4/17.
- */
 
 public class QRCodeUtils {
+    public static final String QR_CODE_FILEPATH = Collect.SETTINGS + File.separator + "collect-settings.jpeg";
+    static final String MD5_CACHE_PATH = Collect.SETTINGS + File.separator + ".md5";
```

Comment:
Maybe we should have a more descriptive name for the QR Code MD5 hash than “.md5”.

Reply:
How about `.collect-settings-hash`?

Reply:
Works for me.

Reply:
:+1: